docs/contributors/native-mobile.md

```diff
@@ -0,0 +1,6 @@
+# React Native based mobile Gutenberg
+Interwined with the web codepaths, the Gutenberg repo also includes the React Native based mobile tree. The mobile client is packaged and released via the [official WordPress apps](https://wordpress.org/mobile/). Even though the build pipeline is rather different and lives in its own repo for now ([here's the native mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile)), the source code itself is taken directly from this repo and the "web" side codepaths.
+## Native mobile specific files
+The majority of the code shared with native mobile is in the very same JavaScript module and SASS style files. In the cases where the code paths need to diverge, a `.native.js` or `.native.scss` variant of the file is created. In some cases, platform specific files can be also found for Android (`.android.js`) or iOS (`.ios.js`).
+## Mind the mobile
+Our tooling is as great yet as we'd like to and it's hard to have good awareness of those native mobile files. That means that contributors need to manually pay attention to update the native mobile files during code refactorings. For example, renaming a function or a prop should also be performed in the native modules too, otherwise the mobile client will break. We are in the process of putting some more safeguards in place in PRs, but we're still far from done. Please bare with us and thank you in advance. ❤️🙇‍
```

Comment:
I assume that you wanted to say that the tooling isn't good enough, feel free to ignore if you think differently :)

```suggestion
Our tooling isn't as good yet as we'd like to and it's hard to have a good awareness of those native mobile files. That means that contributors need to manually pay attention to update the native mobile files during code refactorings. For example, renaming a function or a prop should also be performed in the native modules too, otherwise, the mobile client will break. We are in the process of putting some more safeguards in place in PRs, but we're still far from done. Please bear with us and thank you in advance. ❤️🙇‍
```